app/models/aid_application.rb

```diff
@@ -82,6 +89,7 @@ class AidApplication < ApplicationRecord
     validates :street_address, presence: true
     validates :city, presence: true
     validates :zip_code, presence: true, zip_code: true
+    validates :mailing_zip_code, presence: true, five_digit_zip: true
```

Comment:
The presence validator on mailing_zip_code doesn't seem right if mailing address is optional.